apps/odyssey-stats/src/components/odyssey-query-site-purchases.ts

```diff
@@ -52,7 +52,7 @@ const useOdysseyQuerySitePurchases = ( siteId: number | null ) => {
 	}, [ siteId, reduxDispatch ] );
 };
 
-export default function OdysseyQuerySitePurchases( siteId: number | null ) {
+export default function OdysseyQuerySitePurchases( { siteId }: { siteId: number | null } ) {
```

Comment:
‼️

Reply:
The `siteId` is used to [compare with existing](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/purchases/reducer.js#L67C56-L67C69) fetched purchases, so it worked well in this case. 😅